webapp/package.json

```diff
@@ -5,36 +5,37 @@
   "author": "Mattermost",
   "license": "Apache-2.0",
   "main": "src/index.js",
+  "private": true,
```

Comment:
What does this mean?

Reply:
This mean this package will not be published on NPM. It is mainly to remove warnings about webapp not having a repository defined.

Reply:
It also makes sure you do not do `npm publish` by mistake.